docs/book/src/topics/confidential-vms.md

```diff
@@ -0,0 +1,69 @@
+# Confidential VMs
+
+This document describes how to deploy a cluster with Azure [Confidential VM](https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-vm-overview) nodes.
```

Comment:
```suggestion
This document describes how to deploy a cluster with Azure [Confidential VM](https://learn.microsoft.com/azure/confidential-computing/confidential-vm-overview) nodes.
```

This is nitpicky, but I try to remove the `en-us/` section of URLs so the user can (possibly) land on a localized version of the document. Same for the other URLs in these new docs, but it's not crucial to fix. (We should probably make a comprehensive pass over the docs at some point for this.)